music21/humdrum/spineParser.py

```diff
@@ -1372,12 +1373,14 @@ def processChordEvent(self, eventC):
         # multipleNotes
         notesToProcess = eventC.split()
         chordNotes = []
-        for noteToProcess in notesToProcess:
-            thisNote = hdStringToNote(noteToProcess)
+        defaultDuration = None  # used for non-first notes without a duration
+        for i, noteToProcess in enumerate(notesToProcess):
+            thisNote = hdStringToNote(noteToProcess, defaultDuration)
+            if i == 0:
+                defaultDuration = thisNote.duration
             chordNotes.append(thisNote)
-        eventChord = chord.Chord(chordNotes, beams=chordNotes[-1].beams)
-        eventChord.duration = chordNotes[0].duration
-
+        eventChord = chord.Chord(chordNotes, beams=chordNotes[-1].beams,  # type: ignore
```

Comment:
Can you explain the `type: ignore` here?

Reply:
I had to use type ignore where I couldn't reproduce the music21 type-checking locally and kept failing when pushing even though the types were right. It might have been an issue with things being set to None initially and then replaced with a different type according to some logic. I'll give this typing another try.